cmd/tk/env.go

```diff
@@ -164,14 +166,22 @@ func addEnv(dir string, cfg *v1alpha1.Environment) error {
 	// the other properties are already set by v1alpha1.New() and pflag.Parse()
 	cfg.Metadata.Name, _ = filepath.Rel(rootDir, path)
 
-	// write spec.json
-	if err := writeJSON(cfg, filepath.Join(path, "spec.json")); err != nil {
-		return err
-	}
+	if inline {
+		cfg.Data = struct{}{}
+		// write main.jsonnet with inline tanka.dev/Environment
+		if err := writeJsonnet(cfg, filepath.Join(path, "main.jsonnet")); err != nil {
+			return err
+		}
+	} else {
+		// write spec.json
+		if err := writeJSON(cfg, filepath.Join(path, "spec.json")); err != nil {
+			return err
+		}
 
-	// write main.jsonnet
-	if err := writeJSON(struct{}{}, filepath.Join(path, "main.jsonnet")); err != nil {
-		return err
+		// write main.jsonnet
+		if err := writeJSON(struct{}{}, filepath.Join(path, "main.jsonnet")); err != nil {
```

Comment:
Since we are writing a jsonnet file, should we use `writeJsonnet` here as well for consistency?